test/e2e/lib/poll.bash

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+function poll_until_equals() {
+  local what="$1"
+  local expected="$2"
+  local check_cmd="$3"
+  poll_until_true "$what" "[ '$expected' = \"\$( $check_cmd )\" ]"
+}
+
+function poll_until_true() {
```

Comment:
Can we either add a comment here that this waits for a max of `2m`, or make it more obvious by making it configurable?

I think otherwise we all individually end up making the `24*5` calculation multiple times during the debugging of a (time based) failing test.

Reply:
sure, that makes sense

Reply:
I was waiting for a test needed a different period/total wait time to add the parameters